api/tests/core/search/test_serialize_algolia.py

```diff
@@ -8,6 +8,7 @@
 from pcapi.core.categories import subcategories
 import pcapi.core.offerers.factories as offerers_factories
 import pcapi.core.offerers.models as offerers_models
+from pcapi.core.offers import factories as offers_factories
```

Comment:
(style / question de goût) On importe plutôt comme suit : 

```suggestion
import pcapi.core.offers.factories as offers_factories
```

1. C'est plus facile de grepper le chemin du module complet (`grep core.offers.factories`).
2. Ça prend 5 caractères de moins, ce qui sauve des arbres. Je pense. J'espère ? Au moins une petite branche, peut-être ?

Oui, je te l'accorde, c'est pas mal une question de goût, au final. ;)